iml-gui/crate/src/components/progress_circle.rs

```diff
@@ -24,7 +24,7 @@ pub fn used_to_color(used: f64) -> &'static str {
 pub fn view<'a, T>(x: impl Into<Option<(f64, &'a str)>>) -> Node<T> {
     match x.into() {
         Some((used, color)) => {
-            let stroke_length = (1.0 - used) * CIRCUMFERENCE;
+            let stroke_length = (100.0f64 - used) / 100.0f64 * CIRCUMFERENCE;
```

Comment:
nit: f64 is unnecessary when using a decimal.

Reply:
Yeah I just thought I would be explicit :)